lib/controller.js

```diff
@@ -24,22 +24,40 @@ var responses = {
 
 // Wrapper function for Tessel.list to set SSH key path
 controller.list = function(opts) {
-  return controller.keyHelper(opts, Tessel.list);
+  return controller.defaultHelpers(opts, Tessel.list);
 };
 
 // Wrapper function for Tessel.get to set SSH key path
 controller.get = function(opts) {
-  return controller.keyHelper(opts, Tessel.get);
+  return controller.defaultHelpers(opts, Tessel.get);
 };
 
-controller.keyHelper = function(opts, next) {
+// Calls any helpers for the get and list codepaths
+controller.defaultHelpers = function(opts, next) {
+  // First disable output if necessary
+  return controller.outputHelper(opts)
+    // Then make sure our SSH Key path is updated
+    .then(() => controller.keyHelper(opts))
+    // before continuing to the next function
+    .then(() => next(opts));
+};
+
+controller.keyHelper = function(opts) {
```

Comment:
It was necessary because I wanted to add the function below (`outputHelper`) without making the code look too hairy. `keyHelper` originally set the default SSH key and then was responsible for calling the subsequent command (either `list` or `get`) but I figured it makes more sense for one function to call each of the helpers and then pass it on to the command.